lambda-notes/src/actions/index.js

```diff
@@ -0,0 +1,84 @@
+import axios from 'axios';
+
+export const FETCHING = "FETCHING";
+export const SUCCESS = "SUCCESS";
+export const FAILURE = "FAILURE";
+export const UPDATING = "UPDATING";
+export const UPDATED = "UPDATED";
+export const POSTING = "POSTING";
+export const CREATED = "CREATED";
+export const DELETING = "DELETING";
+export const DELETED = "DELETED";
+
+export const fetch = () => {
+    return dispatch => {
+        dispatch({ type: FETCHING });
+        axios.get("https://fe-notes.herokuapp.com/note/get/all")
+        .then(({data})   =>  {
+            dispatch({ type: SUCCESS, payload: data });
+        })
+        .catch(err  =>  {
+            return dispatch({ type: FAILURE, payload: err })
```

Comment:
Don't need to return dispatch, it's already being returned at the beginning of the function (line 14 in this case), and dispatch already returns a function itself by calling it.